packages/db-postgres/src/types.ts

```diff
@@ -96,5 +117,36 @@ declare module 'payload' {
       }
     }
     tables: Record<string, GenericTable>
+    versionsSuffix?: string
+  }
+}
+
+declare module 'payload/types' {
```

Comment:
Did we verify this module declaration has the intended effect in an editor?

Reply:
This approach DID NOT work. I've removed them https://github.com/payloadcms/payload/pull/4525 and just put the property directly on the payload types.